i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-biztalk.md

```diff
@@ -2,36 +2,298 @@
+2. Renseignez les macros désirées (par exemple, ajustez les seuils d'alerte). Les macros indiquées ci-dessous comme requises (**Obligatoire**) doivent être renseignées.
 
-| Field                                   | Value                      |
-| :-------------------------------------- | :------------------------- |
-| Host name                               | *Name of the host*         |
-| Alias                                   | *Host description*         |
-| IP                                      | *Host IP Address*          |
-| Monitored from                          | *Monitoring Poller to use* |
-| Host Multiple Templates                 | App-Biztalk-custom         |
+<Tabs groupId="sync">
+<TabItem value="Received-Locations-Disabled" label="Received-Locations-Disabled">
 
-Click "Save" button.
+| Macro        | Description                                                                                                                                      | Valeur par défaut | Obligatoire |
+|:-------------|:-------------------------------------------------------------------------------------------------------------------------------------------------|:------------------|:-----------:|
+| WARNING      | Warning threshold                                                                                                                                |                   |             |
+| CRITICAL     | Critical threshold                                                                                                                               |                   |             |
+| EXTRAOPTIONS | Any extra option you may want to add to the command (a --verbose flag for example). Toutes les options sont listées [ici](#options-disponibles). | --verbose         |             |
+
+</TabItem>
+</Tabs>
+
+3. [Déployez la configuration](/docs/monitoring/monitoring-servers/deploying-a-configuration). Le service apparaît dans la liste des services supervisés, et dans la page **Statut des ressources**. La commande envoyée par le connecteur est indiquée dans le panneau de détails du service : celle-ci montre les valeurs des macros.
+
+## Comment puis-je tester le plugin et que signifient les options des commandes ?
+
+Une fois le plugin installé, vous pouvez tester celui-ci directement en ligne
+de commande depuis votre collecteur Centreon en vous connectant avec
+l'utilisateur **centreon-engine** (`su - centreon-engine`). Vous pouvez tester
+que le connecteur arrive bien à superviser une ressource en utilisant une commande
+telle que celle-ci (remplacez les valeurs d'exemple par les vôtres) :
+
+```bash
+/usr/lib/centreon/plugins/centreon_biztalk.pl \
+	--plugin=database::mssql::plugin \
+	--dyn-mode=apps::biztalk::sql::mode::rlocationdisabled \
+	--hostname=10.0.0.1 \
+	--port=1433 \
+	--username='' \
+	--password='' \
```

Comment:
username and password

Reply:
Non obligatoires voir ici : https://github.com/centreon/centreon-documentation/pull/3852/files#r1879596308